lite-variables.tf

```diff
@@ -0,0 +1,194 @@
+  validation {
+    condition     = length(regexall("^[^@]+@[^@]+$", var.aks_admin_email)) > 0
+    error_message = "Value must at least look like a valid email address."
+  }
+}
+
+variable "aks_admin_name" {
+  type        = string
+  description = "REQUIRED.  Name of the administrator of this HPCC Systems cluster.\nExample entry: Jane Doe"
+}
+
+variable "aks_azure_region" {
+  type        = string
+  description = "REQUIRED.  The Azure region abbreviation in which to create these resources.\nMust be one of [\"eastus\", \"eastus2\", \"centralus\"].\nExample entry: eastus2"
+  validation {
+    condition     = contains(["eastus", "eastus2", "centralus"], var.aks_azure_region)
+    error_message = "Value must be one of [\"eastus\", \"eastus2\", \"centralus\"]."
+  }
+}
+
+variable "aks_enable_roxie" {
+  description = "REQUIRED.  Enable ROXIE?\nThis will also expose port 8002 on the cluster.\nExample entry: false"
+  type        = bool
+  default     = false
+}
+
+variable "aks_dns_zone_resource_group_name" {
+  type        = string
+  description = "REQUIRED. Name of the resource group containing the dns zone."
+}
+
+variable "aks_dns_zone_name" {
+  type        = string
+  description = "REQUIRED. dns zone name. The name of existing dns zone."
+}
+
+variable "aks_admin_ip_cidr_map" {
+  description = "OPTIONAL.  Map of name => CIDR IP addresses that can administrate this AKS.\nFormat is '{\"name\"=\"cidr\" [, \"name\"=\"cidr\"]*}'.\nThe 'name' portion must be unique.\nDefault value is '{}' means no CIDR addresses.\nThe corporate network and your current IP address will be added automatically, and these addresses will have access to the HPCC cluster as a user."
+  type        = map(string)
+  default     = {}
+}
+
+variable "aks_node_sizes" {
+  description = "REQUIRED.  The VM size for each node in the HPCC Systems node pool.\nRecommend \"Standard_B4ms\" or better.\nSee https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-general for more information."
+  type = object({
+    roxie           = optional(string, "2xlarge")
+    serv            = optional(string, "4xlarge")
+    spray           = optional(string, "2xlarge")
+    thor            = optional(string, "2xlarge")
+  })
+}
+#===== end of aks variables =====
+
+variable "my_azure_id" {
+  description = "REQUIRED. The id of your azure account."
+  type        = string
+}
+
+variable "external_storage_desired" {
+  description = "If you want external storage instead of ephemeral, this should be true. For ephemeral storage this should be false"
+  type        = bool
+  default     = false
+}
+
+variable "enable_thor" {
+  description = "REQUIRED.  If you want a thor cluster."
+  type        = bool
+  default     = true
+}
+
+variable "a_record_name" {
+  type        = string
+  description = "OPTIONAL: dns zone A record name"
+  default     = ""
+}
+
+variable "admin_username" {
```

Comment:
Curious: Why is there a separate entry for HPCC admin and AKS admin?

Reply:
So admin_name and aks_admin_name are really the same but you're asking the user to explicitly set them both?  Can you set one equal to the value of the other so the user explicitly defines only one?  Then you can remove documentation for the one the user doesn't see.

Reply:
TLH: 'admin_name' doesn't exist any more (this is its name in your terraform). It is now called 'aks_admin_name'. Here is an example of how it would be set 'aks_admin_name="Timothy L Humphrey". I think you are confusing 'admin_username' with 'admin_name'. An example of setting 'admin_username' is as follows: 'admin_username="tlhumphrey2"

Reply:
Argh!  You are right!  I confused those two things.